dstep/driver/Application.d

```diff
@@ -48,16 +46,22 @@ class Application : DStack.Application
 
     protected override void run ()
     {
+        derr.writefln(arguments.rawArgs);
         handleArguments;
 
         if (arguments.argument.input.hasValue)
         {
-            inputFiles ~= arguments.argument.input;
-            startConversion(inputFiles.first);
+            auto inputFiles = arguments.argument.input.values[1 .. $];
+            foreach(string fileName; inputFiles)
```

Comment:
Put a space after the `foreach` keyword. Surround the foreach statement with an empty newline.